packages/grid/devspace.yaml

```diff
@@ -267,6 +269,8 @@ deployments:
                 value: "${VERSION_HASH}"
               - name: NODE_TYPE
                 value: "${NODE_TYPE}"
+              - name: NODE_SIDE_TYPE
+                value: "${NODE_SIDE_TYPE}"
```

Comment:
NIT: We could remove this from frontend as does not use node_side_type env variable

Reply:
Resolved in [b3e40f1](https://github.com/OpenMined/PySyft/pull/8293/commits/b3e40f1d8959443ab0513a61280890be44988d3e)